config/initializers/subscriptions.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+Rails.application.config.after_initialize do
+  # Subscriptions for CollectionsMailer
+  Notifier.subscribe_mailer(event_name: Notifier::DEPOSITOR_ADDED, mailer_class: CollectionsMailer,
+                            mailer_method: :invitation_to_deposit_email)
+  Notifier.subscribe_mailer(event_name: Notifier::MANAGER_ADDED, mailer_class: CollectionsMailer,
+                            mailer_method: :manage_access_granted_email)
```

Comment:
Can you say why the reviewer event doesn't need to be here?

Reply:
Adding reviewer is pending https://github.com/sul-dlss/hungry-hungry-hippo/pull/493